AP-protocol.md

```diff
@@ -354,13 +354,12 @@ pub enum MessageContent {
 	ReqMessageSend { to: NodeId, message: Vec<u8> },
 
 	// Server -> Client
-	RespServerType(ServerType)
+	RespServerType(ServerType),
 	RespFilesList(Vec<u64>),
 	RespFile(Vec<u8>),
 	RespMedia(Vec<u8>),
-	ErrUnsupporedRequestType,
-	ErrRequestedNotFound
-
```

Comment:
I would keep the separation line. Between text and media and chat message. But that may just be me.

Reply:
Ok, didn't notice